cirq-core/cirq/ops/gate_operation_test.py

```diff
@@ -494,6 +494,7 @@ def all_subclasses(cls):
         cirq.Pauli,
         # Private gates.
         cirq.transformers.analytical_decompositions.two_qubit_to_fsim._BGate,
+        cirq.transformers.routing.visualize_routed_circuit._SwapPrintGate,
```

Comment:
Don't include an internal class in the global cirq namespace.

Reply:
It seems like it has to be included or else I get the error
`AssertionError: <class 'cirq.transformers.routing.visualize_routed_circuit._SwapPrintGate'> has no json file, please add a json file or add to the list of classes to be skipped if there is a reason this gate should not round trip to a gate via creating an operation.`